ftw/testing/freezer.py

```diff
@@ -37,6 +39,11 @@ def __enter__(self):
                 'The freeze_date argument must be a datetime.datetime'
                 ' instance, got %s' % type(self.new_now).__name__)
 
+        def is_caller_ignored(frames_up):
+            caller_frame = inspect.getouterframes(inspect.currentframe(), 0)[frames_up][0]
```

Comment:
Use `caller_frame = inspect.stack()[frames_up][0]` instead? That should be exactly equivalent to what you're doing here. I find the [`inspect.stack()`](https://docs.python.org/3/library/inspect.html#inspect.stack) variant to be much more readable to be honest.

Reply:
Indeed, updated 👍